assets/css/style.css

```diff
@@ -3,9 +3,322 @@
     margin: 0;
     padding: 0;
     box-sizing: border-box;
+    font-family: 'Kumbh Sans', sans-serif;;
```

Comment:
Importuj prvo font. Nisi uopste importovao "Kumbh Sans" font pa ti se prikazuje fallback font, sans-serif. To mozes da uradis u index.html ili ovde u css-u

Reply:
Vazi.

Reply:
Okej, je l' samo to nije u redu?

Reply:
Da, promeni to vezano za fontove i proveri da li je responsive ok na svim velicinama ekrana, portrait i landscape

Reply:
Landscape je u redu na 350px i 1440px